isis/src/base/objs/TrackingTable/TrackingTable.h

```diff
@@ -0,0 +1,67 @@
+#ifndef TrackingTable_h
+#define TrackingTable_h
+/**
+ * @file
+ * $Revision: 1.3 $
+ * $Date: 2010/05/14 19:17:09 $
+ *
+ *   Unless noted otherwise, the portions of Isis written by the USGS are public
+ *   domain. See individual third-party library and package descriptions for
+ *   intellectual property information,user agreements, and related information.
+ *
+ *   Although Isis has been used by the USGS, no warranty, expressed or implied,
+ *   is made by the USGS as to the accuracy and functioning of such software
+ *   and related material nor shall the fact of distribution constitute any such
+ *   warranty, and no responsibility is assumed by the USGS in connection
+ *   therewith.
+ *
+ *   For additional information, launch
+ *   $ISISROOT/doc//documents/Disclaimers/Disclaimers.html in a browser or see
+ *   the Privacy &amp; Disclaimers page on the Isis website,
+ *   http://isis.astrogeology.usgs.gov, and the USGS privacy and disclaimers on
+ *   http://www.usgs.gov/privacy.html.
+ */
+
+#include "FileName.h"
+#include "Pvl.h"
+#include "Table.h"
+
+#include <QList>
+#include <QString>
+
+
+namespace Isis {
+  /**
+   * @brief 
+   * @author 2004-09-01 Jeff Anderson
+   *
+   * @internal
+   *   @history 2005-03-18 Elizabeth Ribelin - Added documentation to class
+   */
+  class TrackingTable{
+    
+    public:
+      
+      static QString tableName = "InputImages";
```

Comment:
This needs to be moved outside of the class and we should rename it to something like trackingTableName because then it will be accessible as "Isis::trackingTableName" similar to how special pixel values can be accessed.

It also needs to be const not static, my bad!

Reply:
See SpecialPixel.h for how this should be done.